apps/backend/src/controllers/neighborhoods.ts

```diff
@@ -105,4 +107,102 @@ neighborhoodsRouter.post('/:id/join', middleware.userIdExtractorAndLoginValidato
   return res.status(201).send({ success: 'You have joined the neighborhood' });
 }));
 
+// Get a neighborhood's requests
+neighborhoodsRouter.get('/:id/requests', middleware.userIdExtractorAndLoginValidator, catchError(async (req: RequestWithAuthentication, res: Response) => {
+  const neighborhoodID = Number(req.params.id);
+  // LoginValidator ensures that loggedUserId is present
+  const loggedUserID = req.loggedUserId as number;
+
+  const isUserMemberOfNeighborhood: boolean = await neighborhoodServices
+    .isUserMemberOfNeighborhood(loggedUserID, neighborhoodID);
+
+  if (!isUserMemberOfNeighborhood) {
+    return res.status(400).send({ error: 'user is not a member of the neighborhood' });
+  }
+
+  const requests: RequestData[] = await neighborhoodServices
+    .getRequestsAssociatedWithNeighborhood(neighborhoodID);
+
+  return res.status(200).send(requests);
+}));
+
+// Get a single request
+neighborhoodsRouter.get('/:neighborhoodId/requests/:requestId', middleware.userIdExtractorAndLoginValidator, catchError(async (req: RequestWithAuthentication, res: Response) => {
```

Comment:
I think this can work as just `/requests/:id` and be made a part of requests.ts

Reply:
How do we pass neighborhoodId then?

Reply:
We don't need to pass neighborhood id. It's already one of the fields in the request table if we need it.

Reply:
> I think this can work as just `/requests/:id` and be made a part of requests.ts

That is my thinking as well.